discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -250,6 +250,18 @@ def wait_for_hosts_passed_reboot(self, nodes_count=env_variables['num_nodes']):
             nodes_count=nodes_count
         )
 
+    def wait_for_hosts_passed_reboot(self, nodes_count=env_variables['num_nodes']):
```

Comment:
Let's make the previous function with another parameter `hasStagePassed: bool = False` instead of having two functions.